src/BTVNanoCommissioning/workflows/ttdilep_valid_sf.py

```diff
@@ -195,7 +230,7 @@ def process(self, events):
         event_jet = events.Jet[
             (events.Jet.pt > 25)
             & (abs(events.Jet.eta) <= 2.4)
-            # & (events.Jet.puId > 0) # commented out due to run3 condition
+            & (events.Jet.puId > 0)  # commented out due to run3 condition
```

Comment:
Could you pack this in a condition that puId is only requested for non-Run3 campaigns?

Reply:
Yes, then I will do it for all the workflows